R/connect.R

```diff
@@ -59,7 +59,7 @@ connectClient <- function(service, authInfo) {
     },
 
     currentUser = function() {
-      handleResponse(GET(service, authInfo, "/users/current/"))
+      handleResponse(GET(service, authInfo, "/users/current"))
```

Comment:
We do, but it doesn't seem to be working quite right, which I'm looking into. But in any case, it makes no sense to make an API call, only to be redirected without a slash. Trying to save a hop here.